common/utils.c

```diff
@@ -223,3 +224,15 @@ int tmpdir_mkstemp(const tal_t *ctx, const char *template TAKES, char **created)
 
 	return fd;
 }
+
+void err_1(const char *fmt, ...)
+{
```

Comment:
This is called errx, it already exists in ccan/err, and I don't see where you call it?

Reply:
It's used in the previous patch.. will remove this one and update that one to use `errx`